packages/superset-ui-preset-chart-xy/src/utils/XYChartLayout.tsx

```diff
@@ -154,7 +154,7 @@ export default class XYChartLayout {
 
     return axis && this.xLayout ? (
       <XAxis
-        label={axis.getTitle()}
+        label={axis.getTitle() || ''}
```

Comment:
Why not have getTitle return `''` instead of undefined, and then have `hasTitle` check against that instead of undefined?

Because we want different behavior for a user defined empty title?